workflow/validate/lint.go

```diff
@@ -6,6 +6,7 @@ import (
 	"path/filepath"
 
 	"github.com/argoproj/pkg/json"
```

Comment:
Are we going to need this guy too?

Reply:
not unless we need to modify it https://github.com/argoproj/pkg